### DISCUSSION TITLE
SCRUM: Reunión 30-08-2023

### DISCUSSION THREAD

**Discussion by RamiroV13** [adempiere/adempiere#4144] [General]
<!DOCTYPE html>
<html>

<head>
    <title></title>
</head>

<body>
    
    <h2 dir="auto" style='box-sizing: border-box; margin-top: 24px; margin-bottom: 16px; font-size: 1.5em; font-weight: var(--base-text-weight-semibold, 600); line-height: 1.25; padding-bottom: 0.3em; border-bottom: 1px solid var(--borderColor-muted, var(--color-border-muted)); color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>Datos de la Reuni&oacute;n</h2>
    <p dir="auto" style='box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Moderador:</strong><span>&nbsp;</span>Alberto Abudinen<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Secretario:</strong><span>&nbsp;</span><br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Fecha:</strong><span>&nbsp;</span>30 de Agosto 2023<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Hora de Inicio:</strong><span>&nbsp;</span>15:00 pm ~ 16:00 pm<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Lugar:</strong><span>&nbsp;</span>Discord<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Objetivo:</strong><span>&nbsp;</span>Metodolog&iacute;a de Trabajo</p>
    <h2 dir="auto" style='box-sizing: border-box; margin-top: 24px; margin-bottom: 16px; font-size: 1.5em; font-weight: var(--base-text-weight-semibold, 600); line-height: 1.25; padding-bottom: 0.3em; border-bottom: 1px solid var(--borderColor-muted, var(--color-border-muted)); color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>Participantes:</h2>
    <ul dir="auto" style='box-sizing: border-box; padding-left: 2em; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>
        <li style="box-sizing: border-box; margin-left: 0px;">@e-Evolution</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">@marcalwestf</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">@yamelsenih</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">@eduardolopezuy<a class="user-mention notranslate" data-hovercard-type="user" data-hovercard-url="/users/niclopup/hovercard" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="https://github.com/niclopup" style="box-sizing: border-box; background-color: transparent; color: var(--color-fg-default); text-decoration: none; font-weight: var(--base-text-weight-semibold, 600); white-space: nowrap;"></a></li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">@niclopup</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">@RamiroV13</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">@AlbertoAbudinen</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">@piracio</li>
    </ul>
    <p dir="auto" style='box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'><br></p>
    <h2 dir="auto" style='box-sizing: border-box; margin-top: 24px; margin-bottom: 16px; font-size: 1.5em; font-weight: var(--base-text-weight-semibold, 600); line-height: 1.25; padding-bottom: 0.3em; border-bottom: 1px solid var(--borderColor-muted, var(--color-border-muted)); color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>Metodolog&iacute;a de Trabajo</h2>
    <ul dir="auto" style='box-sizing: border-box; padding-left: 2em; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>
        <li style="box-sizing: border-box; margin-left: 0px;">Se definen trabajos realizados tanto como t&eacute;cnicos como funcionales en ambiente ADempiere 394 tanto como para ciudadanos</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">Se plantean vias para el cobro monetario de los citizens como Stripe</li>
    </ul>
    <h2 dir="auto" style='box-sizing: border-box; margin-top: 24px; margin-bottom: 16px; font-size: 1.5em; font-weight: var(--base-text-weight-semibold, 600); line-height: 1.25; padding-bottom: 0.3em; border-bottom: 1px solid var(--borderColor-muted, var(--color-border-muted)); color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>Acuerdos</h2>
    <ol dir="auto" style='box-sizing: border-box; padding-left: 2em; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>
        <li style="box-sizing: border-box; margin-left: 0px;">V&iacute;ctor mandar&aacute; un segundo listado de ciudadanos (citizens) para as&iacute; poder homologarlos al ambiente ADempiere 394 ya previamente levantado por V&iacute;ctor</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">Se llega a un acuerdo con todos los presentes en la reuni&oacute;n para definir el inicio del &quot;Bug Fix Day&quot; para el viernes 01 de Septiembre 2023 9:00 AM (GMT-3)</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">OFB realizar&aacute; una segunda carga de ciudadanos (citizens) en ambiente 394</li>
        <li style="box-sizing: border-box; margin-top: 0.25em; margin-left: 0px;">Victor y Horacio configuraran plantilla de correo y Stripe para el cobro mensual/anual de los ciudadanos (citizens)</li>
    </ol>
    <h2 dir="auto" style='box-sizing: border-box; margin-top: 24px; margin-bottom: 16px; font-size: 1.5em; font-weight: var(--base-text-weight-semibold, 600); line-height: 1.25; padding-bottom: 0.3em; border-bottom: 1px solid var(--borderColor-muted, var(--color-border-muted)); color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>Compromisos</h2>
    <p>&Iacute;tem | Responsables | Fecha del acuerdo | Estado del Compromiso -- | -- | -- | -- Envio segunda lista ciudadanos | @e-Evolution | 30/08/2023 | Pendiente Cargar ciudadanos en ADempiere 394 | @RamiroV13 | 30/08/2023 | Pendiente Inicio Bug fix day | Todos | 30/08/2023 | Pendiente Configuraci&oacute;n plantilla correo y Stripe | @e-Evolution @piracio | 30/08/2023 | Pendiente</p>
    <h2 dir="auto" style='box-sizing: border-box; margin-top: 24px; margin-bottom: 16px; font-size: 1.5em; font-weight: var(--base-text-weight-semibold, 600); line-height: 1.25; padding-bottom: 0.3em; border-bottom: 1px solid var(--borderColor-muted, var(--color-border-muted)); color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'>Pr&oacute;xima reuni&oacute;n</h2>
    <p dir="auto" style='box-sizing: border-box; margin-top: 0px; margin-bottom: 0px !important; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji"; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;'><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Fecha:</strong><span>&nbsp;6</span> de Septiembre 2023<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Hora de Inicio:</strong><span>&nbsp;</span>15:00 pm ~ 16:00 pm<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Lugar:</strong><span>&nbsp;</span>Discord<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Objetivo:</strong><span>&nbsp;Avances y Planificaci&oacute;n proyecto ADempiere</span></p>
    
</body>

</html>

**Comment by piracio** [comment]
Hola, @RamiroV13 no salgo como participante :(

**Reply by RamiroV13** [reply]
Ahora si! Estoy puliendo la minuta ya va tomando forma :)
